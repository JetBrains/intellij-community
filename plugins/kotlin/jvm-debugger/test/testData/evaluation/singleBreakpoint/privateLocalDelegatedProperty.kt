class Main {
    fun foo(): Int {
        //Breakpoint!
        return 42
    }
    private val privateDelegatedProperty: Int by lazy { 24 }
}

fun main() {
    Main().foo()
}

// EXPRESSION: privateDelegatedProperty
// RESULT: 24: I