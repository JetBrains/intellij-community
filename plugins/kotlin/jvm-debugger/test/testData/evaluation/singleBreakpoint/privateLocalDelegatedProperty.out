LineBreakpoint created at privateLocalDelegatedProperty.kt:4
Run Java
Connected to the target VM
privateLocalDelegatedProperty.kt:4
Compile bytecode for privateDelegatedProperty
Disconnected from the target VM

Process finished with exit code 0
